README.md

```diff
@@ -19,94 +19,206 @@
 
 [![tweet][tweet-badge]][tweet]
 
+> [!TIP]
+> Download a free copy of an ebook that goes step-by-step through [incremental React Native adoption in native apps](https://www.callstack.com/ebooks/incremental-react-native-adoption-in-native-apps?utm_campaign=brownfield&utm_source=github&utm_medium=referral&utm_content=react-native-brownfield).
```

Comment:
Maybe we could get an ebook banner? This can be easily missed

Reply:
hope this works? 
<img width="1090" height="682" alt="image" src="https://github.com/user-attachments/assets/22b7f2c5-9aae-420b-b794-b02443f42512" />